.github/scripts/Tests/render-portfolio.Tests.ps1

```
@@ -1190,4 +1190,299 @@ rounds:
+  - lane: main
+    round: 1
+    issues: [425, 571]
+'@ | Set-Content -Path $tempSpecCeil1 -Encoding UTF8
+
+        try {
+            try {
+                Invoke-PortfolioRender -specPath $tempSpecCeil1 -issueScanLimit 3
+            }
+            catch {
+                # Expected: ceiling guard throws; swallow so we can assert below.
+            }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)